libats-slick/src/main/scala/com/advancedtelematic/libats/slick/db/SlickExtensions.scala

```diff
@@ -29,6 +29,15 @@ object SlickPipeToUnit {
 
 trait SlickResultExtensions {
   implicit class DBIOActionExtensions[T](action: DBIO[T]) {
+
+    def handleForeignKeyError(error: Throwable)(implicit ec: ExecutionContext): DBIO[T] = {
```

Comment:
Why is not it a part of `handleIntegrityErrors`?

Reply:
With the new FK `campaigns -> updates` I thought it would be nice to have a different error message when there is a FK integrity error, instead of the generic one we have now with `handleIntegrityErrors`.
I mean here: https://github.com/advancedtelematic/campaigner/blob/d68340eabab51eb086a51a639ca1fefd0ba00023/src/main/scala/com/advancedtelematic/campaigner/db/Repository.scala#L216